src/components/Project/Project.test.tsx

```diff
@@ -19,7 +19,15 @@ describe("<Project />", () => {
     // project title
     expect(allColumnLabels[0]).toHaveProperty("value", projectTitle);
     // project's star icon
+    expect(screen.getByTestId("boardButton")).toBeTruthy();
     expect(screen.getByTestId("starButton")).toBeTruthy();
+    expect(screen.getByTestId("testButton")).toBeTruthy();
+    expect(screen.getByTestId("workSpaceButton")).toBeTruthy();
+    expect(screen.getByTestId("shareButton")).toBeTruthy();
+    expect(screen.getByTestId("powerUpsButton")).toBeTruthy();
+    expect(screen.getByTestId("automationButton")).toBeTruthy();
+    expect(screen.getByTestId("filterButton")).toBeTruthy();
+    expect(screen.getByTestId("showMenuButton")).toBeTruthy();
```

Comment:
It's not practical to entirely depend on testId when you do UI-focused unit tests. Because it's less close to a real matter you would like to know. For instance, if you wanted to see if a button is correctly rendered, you should test that with `text` or `label` so that testing-library would look for string element instead of a JSX element with the test id, which might not have a string element within.(It's a little chance)

She explains pretty good the reason why: https://betterprogramming.pub/why-you-should-avoid-testing-react-components-with-test-ids-ee50d20d37d2